lib/core/src/Cardano/Wallet.hs

```diff
@@ -698,14 +697,14 @@ restoreStep ctx wid (localTip, (nodeTip, nodeHeight)) = do
         Right (blockFirst : blocksRest) -> do
             let blocks = blockFirst :| blocksRest
             let nextLocalTip = view #header . NE.last $ blocks
-            let measuredTip = (nodeTip ^. #slotId, nodeHeight)
+            let measuredTip = (nodeTip ^. #slotId, nodeTip ^. #blockHeight)
             let action = restoreBlocks @ctx @s @t @k ctx  wid blocks measuredTip
```

Comment:
This is now redundant right? I'd suggest to only pass the `nodeTip` here as an argument for `restoreBlocks`, instead of duplicating the information.